Source/WebCore/loader/PingLoader.cpp

```diff
@@ -157,6 +157,10 @@ void PingLoader::sendViolationReport(Frame& frame, const URL& reportURL, Ref<For
 {
     ASSERT(frame.document());
 
+    // FIXME: Add the concept of browsing context group like in the specification instead of treating the whole process as a group.
```

Comment:
We should file a bugzilla (and Radar) for this topic, and reference it in the FIXME